integreat_cms/cms/templatetags/settings_tags.py

```diff
@@ -28,3 +28,15 @@ def get_base_url():
     :rtype: str
     """
     return settings.BASE_URL
+
+
+@register.simple_tag
+def get_internal_urls():
+    """
+    This tag returns the stringified list of url names
+    which should be treated as internal
+
+    :return: stringified list of internal url names
+    :rtype: str
+    """
+    return settings.INTERNAL_URLS
```

Comment:
```suggestion
    return " ".join(settings.INTERNAL_URLS)
```
When used in a data attribute, I guess it's fine to split by spaces.